app/components/site_search_component/site_search_component.scss

```diff
@@ -0,0 +1,36 @@
+.site-search-component {
+  width: 100%;
+
+  display: flex;
+
+  border: solid 2px $black;
+
+  input {
+    flex-grow: 1;
+    font-family: Roboto;
+    font-size: 19px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 140%;
+
+    padding-left: 1rem;
+
+    border: none;
+    outline: none;
+  }
+
+  button {
+    width: 40px;
+    height: 40px;
+
+    border: none;
+
+    background-image: url('../images/icons/search.svg');
+    background-color: transparent;
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-color: $lime-green;
+
```

Comment:
Could you add a 'cursor: pointer' in here please to give users a hint that it's clickable. We may want to ask for hover state designs too.